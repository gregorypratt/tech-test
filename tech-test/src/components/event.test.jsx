import React from 'react';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';
import Event from './event';

describe('Event', () => {
  const props = {
    competitors: [
      { name: 'Team A', position: 'home' },
      { name: 'Team B', position: 'away' }
    ],
    startTime: '2017-09-19T11:35:23.000Z'
  };

  it('renders without crashing', () => {
    const component = shallow(<Event />);
    expect(shallowToJson(component)).toMatchSnapshot();
  });

  it('displays teams and start time correctly', () => {
    const component = shallow(<Event {...props} />);
    expect(shallowToJson(component)).toMatchSnapshot();
  });
});