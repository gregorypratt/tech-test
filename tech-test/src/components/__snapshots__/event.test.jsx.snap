// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Event displays teams and start time correctly 1`] = `
<div
  className="o-grid"
>
  <div
    className="o-grid__cell o-grid__cell--width-15 u-text--quiet"
  >
    12:35
  </div>
  <div
    className="o-grid__cell"
  >
    <span
      className="u-text--loud"
    >
      Team A
    </span>
    <span
      className="u-text--quiet u-pillar-box-small"
    >
      v
    </span>
    <span
      className="u-text--loud"
    >
      Team B
    </span>
  </div>
</div>
`;

exports[`Event renders without crashing 1`] = `""`;
